doc/book/admin/monitoring/getting_started.rst

```diff
@@ -0,0 +1,105 @@
+..  _monitoring-getting_started:
+
+Getting started with monitoring
+===============================
+
+Example on GitHub: `sharded_cluster_crud_metrics <https://github.com/tarantool/doc/tree/latest/doc/code_snippets/snippets/sharding/instances.enabled/sharded_cluster_crud_metrics>`_
+
+Tarantool allows you to configure and expose its :ref:`metrics <metrics-reference>` using a :ref:`YAML configuration <configuration>`.
+You can also use the :ref:`metrics <metrics-api_reference>` module to create and collect custom metrics.
+
+
+
+
+.. _monitoring_configuring_metrics:
+
+Configuring metrics
+-------------------
+
+To configure metrics, use the :ref:`metrics <configuration_reference_metrics>` section in a cluster configuration.
+The configuration below enables all metrics, excluding :ref:`Vinyl <metrics-reference-vinyl>`-specific ones:
+
+..  literalinclude:: /code_snippets/snippets/sharding/instances.enabled/sharded_cluster_crud_metrics/config.yaml
+    :start-at: metrics:
+    :end-at: instance_name
+    :language: yaml
+    :dedent:
+
+The ``metrics.labels`` option accepts the predefined :ref:`{{ instance_name }} <configuration_predefined_variables>` variable to use an instance name as a :ref:`label <metrics-api_reference-labels>` to be added to every observation.
+
+Third-party Lua modules, like `crud <https://github.com/tarantool/crud>`_ or `expirationd <https://github.com/tarantool/expirationd>`_, offer their own metrics.
+You can enable these metrics by :ref:`configuring the corresponding role <configuration_application_roles_configure>`.
+The example below shows how to enable statistics on called operations by providing the ``roles.crud-router`` role's configuration:
+
+..  literalinclude:: /code_snippets/snippets/sharding/instances.enabled/sharded_cluster_crud_metrics/config.yaml
+    :language: yaml
+    :start-after: routers:
+    :end-at: stats_quantiles
+    :dedent:
+
+``expirationd`` metrics can be enabled as follows:
+
+..  code-block:: yaml
+
+    expirationd:
+      cfg:
+        metrics: true
+
+
+
+..  _monitoring_exposing_metrics:
+
+Exposing metrics
+----------------
+
+To expose metrics in different formats, you can use a third-party `metrics-export-role <https://github.com/tarantool/metrics-export-role>`__ role.
```

Comment:
I wonder if `third-party` is correct here. It look like an official Tarantool component, judging by the GitHub repo. The role may not be available out of the box, but it's from the Tarantool _party_.